### PR TITLE
update doc to match config default value with TiKV codebase.

### PR DESCRIPTION
**Pull request by ZhenhanGong**
Signed-off-by: Zhenhan Gong <zhenhan.gong@gmail.com>

### What is changed, added or deleted? (Required)

Match TiKV docs's config default value with TiKV codebase.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

**Comment by TomShawn** [comment]
@ZhenhanGong Please help confirm all applicable versions. If this PR also applies to other versions, is there any version-specific changes?

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -40,13 +40,13 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `grpc-memory-pool-quota`
 
 + gRPC 可使用的内存大小限制。
-+ 默认值: 32G
++ 默认值: isize::MAX
```
Comment (sticnarf) [inline tikv-configuration-file.md:43]: I prefer to describe this as “无限制”。`isize::MAX` is not readable for users.
TiKV only supports 64-bit platforms, `isize::MAX` is about 8 EB, which is essentially no limit.

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -40,13 +40,13 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
 + 建议仅在出现内存不足 (OOM) 的情况下限制内存使用。需要注意，限制内存使用可能会导致卡顿。
 
 ### `grpc-raft-conn-num`
 
 + tikv 节点之间用于 raft 通讯的链接最大数量。
-+ 默认值：10
++ 默认值：1
```
Comment (sticnarf) [inline tikv-configuration-file.md:49]: This seems to be `1` since 3.0. cc @TomShawn

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -143,24 +143,24 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `use-unified-pool`
 
 + 是否使用统一的读取线程池（在 [`readpool.unified`](#readpoolunified) 中配置）处理存储请求。该选项值为 false 时，使用单独的存储线程池。通过本节 (`readpool.storage`) 中的其余配置项配置单独的线程池。
-+ 默认值：true
++ 默认值：false
```
Comment (sticnarf) [inline tikv-configuration-file.md:146]: I think it's `true` in 5.0.

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -248,13 +248,13 @@ Coprocessor 线程池中线程的栈大小，默认值：10，单位：KiB|MiB|G
 ### `scheduler-concurrency`
 
 + scheduler 内置一个内存锁机制，防止同时对一个 key 进行操作。每个 key hash 到不同的槽。
-+ 默认值：2048000
++ 默认值：512 * 1024
```
Comment (sticnarf) [inline tikv-configuration-file.md:251]: What about putting the calculated value here?

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -248,13 +248,13 @@ Coprocessor 线程池中线程的栈大小，默认值：10，单位：KiB|MiB|G
...
 + 最小值：1
 
 ### `scheduler-worker-pool-size`
 
 + scheduler 线程个数，主要负责写入之前的事务一致性检查工作。
-+ 默认值：4
++ 默认值：cpu_num >= 16 { 8 } else { 4 }
```
Comment (sticnarf) [inline tikv-configuration-file.md:257]: @TomShawn  Is it a preferred format?

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -712,7 +699,7 @@ rocksdb 相关的配置项。
 ### `stats-dump-period`
 
 + 开启 Pipelined Write 的开关。
```
Comment (sticnarf) [inline tikv-configuration-file.md:701]: The description seems not related?

File: tikv-configuration-file.md [hunk 7]
```diff
@@ -1157,7 +1144,7 @@ raftdb 相关配置项。
 ### `max-background-jobs`
 
 + RocksDB 后台线程个数。
-+ 默认值：2
++ 默认值：4
 + 最小值：1
```
Comment (sticnarf) [inline tikv-configuration-file.md:1148]: This is also 2?

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -328,13 +328,6 @@ raftstore 相关的配置项。
 + 默认值：0
 + 最小值：0
 
-### `raft-max-size-per-message`
```
Comment (sticnarf) [inline tikv-configuration-file.md:331]: Why is this removed?

File: tikv-configuration-file.md (hunk 3 above)
Comment (ZhenhanGong) [reply tikv-configuration-file.md:146]: I check tikv code in src/config.rs, seems it is set to None. I assume None is false?
image

Comment (sticnarf) [reply tikv-configuration-file.md:146]: Ah, It's a bit complicated. It should be changed to be the same as the coprocessor one:
```
默认值：如果本节 (`readpool.storage`) 中没有其他配置，默认为 true。否则，为了升级兼容性，默认为 false，请根据需要更改 [`readpool.unified`](#readpoolunified) 中的配置后再启用该选项。
```

File: tikv-configuration-file.md (hunk 7 above)
Comment (ZhenhanGong) [reply tikv-configuration-file.md:1148]: According to Tikv code, it's set to 4.
Screen Shot 2021-03-05 at 14 29 04
Screen Shot 2021-03-05 at 14 29 53

Comment (sticnarf) [reply tikv-configuration-file.md:1148]: I mean the minimal size. Due to `cmp::max(2, ...)`, I think it's actually 2.

File: tikv-configuration-file.md (hunk 8 above)
Comment (ZhenhanGong) [reply tikv-configuration-file.md:331]: I'm not sure whether this config exists or not. 
I didn't find it in Tikv: etc/config-template.toml.
I alse search TiKV codebase for raft_max_size_per_message, still not found.

Comment (ZhenhanGong) [reply tikv-configuration-file.md:331]: Oh, I found it. This config is actually named as "raft_max_size_per_msg".

File: tikv-configuration-file.md (hunk 5 above)
Comment (TomShawn) [reply tikv-configuration-file.md]: No. We usually give a definite number, not a format that only technical personnel can understand. @ZhenhanGong

File: tikv-configuration-file.md [hunk 9]
```diff
@@ -1180,7 +1173,7 @@ raftdb 相关配置项。
 + CA 文件路径
 + 默认值：""
 
-### `cert-path`
+### `cert-path`：：
```
Comment (sticnarf) [inline tikv-configuration-file.md:1176]: Is it a mistake?

File: tikv-configuration-file.md [hunk 10]
```diff
@@ -248,12 +248,12 @@ Coprocessor 线程池中线程的栈大小，默认值：10，单位：KiB|MiB|G
...
 
 + scheduler 内置一个内存锁机制，防止同时对一个 key 进行操作。每个 key hash 到不同的槽。
-+ 默认值：2048000
++ 默认值：524288
 + 最小值：1
 
 ### `scheduler-worker-pool-size`
 
-+ scheduler 线程个数，主要负责写入之前的事务一致性检查工作。
++ scheduler 线程个数，主要负责写入之前的事务一致性检查工作。如果CPU核心数量超过16，默认为8，否则默认为4。
```
Comment (TomShawn) [inline tikv-configuration-file.md:256]:
```suggestion
+ scheduler 线程个数，主要负责写入之前的事务一致性检查工作。如果 CPU 核心数量超过 16，默认为 8；否则默认为4。
```

File: tikv-configuration-file.md [hunk 11]
```diff
@@ -143,24 +143,24 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
 
 + 是否使用统一的读取线程池（在 [`readpool.unified`](#readpoolunified) 中配置）处理存储请求。该选项值为 false 时，使用单独的存储线程池。通过本节 (`readpool.storage`) 中的其余配置项配置单独的线程池。
-+ 默认值：true
++ 默认值：如果本节 (`readpool.storage`) 中没有其他配置，默认为 true。否则，为了升级兼容性，默认为 false，请根据需要更改 [`readpool.unified`](#readpoolunified) 中的配置后再启用该选项。
 
 ### `high-concurrency`
 
 + 处理高优先级读请求的线程池线程数量。
-+ 默认值：4
++ 默认值：cpu_num * 0.5
```
Comment (sticnarf) [inline tikv-configuration-file.md:151]: 精确地来说，应该是 8 ≤ cpu num ≤ 16 时 为 cpu_num * 0.5，cpu num < 8 时为 4，cpu num > 16 时为 8

File: tikv-configuration-file.md [hunk 12]
```diff
@@ -143,24 +143,24 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
 + 最小值：1
 
 ### `normal-concurrency`
 
 + 处理普通优先级读请求的线程池线程数量。
-+ 默认值：4
++ 默认值：cpu_num * 0.5
```
Comment (sticnarf) [inline tikv-configuration-file.md:157]: 同上

File: tikv-configuration-file.md [hunk 13]
```diff
@@ -143,24 +143,24 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
 + 最小值：1
 
 ### `low-concurrency`
 
 + 处理低优先级读请求的线程池线程数量。
-+ 默认值：4
++ 默认值：cpu_num * 0.5
```
Comment (sticnarf) [inline tikv-configuration-file.md:163]: 同上

File: tikv-configuration-file.md [hunk 14]
```diff
@@ -248,12 +248,12 @@ Coprocessor 线程池中线程的栈大小，默认值：10，单位：KiB|MiB|G
...
 
 + scheduler 内置一个内存锁机制，防止同时对一个 key 进行操作。每个 key hash 到不同的槽。
-+ 默认值：2048000
++ 默认值：524288
 + 最小值：1
 
 ### `scheduler-worker-pool-size`
 
-+ scheduler 线程个数，主要负责写入之前的事务一致性检查工作。
++ scheduler 线程个数，主要负责写入之前的事务一致性检查工作。如果 CPU 核心数量超过 16，默认为 8；否则默认为4。
```
Comment (sticnarf) [inline tikv-configuration-file.md:256]: 超过 -> 大于等于

**Review by sticnarf — COMMENTED** [review]
rest lgtm

File: tikv-configuration-file.md (hunk 14 above)
Comment (yikeke) [reply tikv-configuration-file.md:256]:
```suggestion
+ scheduler 线程个数，主要负责写入之前的事务一致性检查工作。如果 CPU 核心数量大于等于 16，默认为 8；否则默认为4。
```

File: tikv-configuration-file.md (hunk 11 above)
Comment (yikeke) [reply tikv-configuration-file.md:151]:
```suggestion
+ 当 8 ≤ cpu num ≤ 16 时，默认值为 cpu_num * 0.5；当 cpu num < 8 时，默认值为 4；当 cpu num > 16 时，默认值为 8。
```

File: tikv-configuration-file.md (hunk 12 above)
Comment (yikeke) [reply tikv-configuration-file.md:157]:
```suggestion
+ 当 8 ≤ cpu num ≤ 16 时，默认值为 cpu_num * 0.5；当 cpu num < 8 时，默认值为 4；当 cpu num > 16 时，默认值为 8。
```

File: tikv-configuration-file.md (hunk 13 above)
Comment (yikeke) [reply tikv-configuration-file.md:163]:
```suggestion
+ 当 8 ≤ cpu num ≤ 16 时，默认值为 cpu_num * 0.5；当 cpu num < 8 时，默认值为 4；当 cpu num > 16 时，默认值为 8。
```

File: tikv-configuration-file.md (hunk 2 above)
Comment (yikeke) [reply tikv-configuration-file.md:49]: Please help update 3.1 and 3.0 versions after this PR is merged? @TomShawn

Comment (TomShawn) [reply tikv-configuration-file.md:49]: OK

**Comment by TomShawn** [comment]
Follow-up: https://github.com/pingcap/docs-cn/pull/5669/files#r603843190

File: tikv-configuration-file.md [hunk 15]
```diff
@@ -248,12 +248,12 @@ Coprocessor 线程池中线程的栈大小，默认值：10，单位：KiB|MiB|G
...
 
 + scheduler 内置一个内存锁机制，防止同时对一个 key 进行操作。每个 key hash 到不同的槽。
-+ 默认值：2048000
++ 默认值：524288
 + 最小值：1
 
 ### `scheduler-worker-pool-size`
 
-+ scheduler 线程个数，主要负责写入之前的事务一致性检查工作。
++ scheduler 线程个数，主要负责写入之前的事务一致性检查工作。如果 CPU 核心数量大于等于 16，默认为 8；否则默认为4。
```
Comment (yikeke) [inline tikv-configuration-file.md:256]:
```suggestion
+ scheduler 线程个数，主要负责写入之前的事务一致性检查工作。如果 CPU 核心数量大于等于 16，默认为 8；否则默认为 4。
```

File: tikv-configuration-file.md [hunk 16]
```diff
@@ -723,8 +717,7 @@ rocksdb 相关的配置项。
 
 ### `stats-dump-period`
 
-+ 开启 Pipelined Write 的开关。
```
Comment (yikeke) [inline tikv-configuration-file.md:726]: 这个解释不用删掉吧？ @sticnarf

**Review by yikeke — APPROVED** [review]
rest LGTM

File: tikv-configuration-file.md (hunk 16 above)
Comment (sticnarf) [reply tikv-configuration-file.md:726]: 这个解释和配置不对应。需要改成 将统计信息输出到日志中的间隔时间

File: tikv-configuration-file.md [hunk 17]
```diff
@@ -723,8 +717,7 @@ rocksdb 相关的配置项。
...
-+ 默认值：true
++ 默认值：10m
```
Comment (yikeke) [inline tikv-configuration-file.md:720]:
```suggestion
+ 将统计信息输出到日志中的间隔时间。
+ 默认值：10m
```

File: tikv-configuration-file.md (hunk 16 above)
Comment (yikeke) [reply tikv-configuration-file.md:726]: 改了

File: tikv-configuration-file.md [hunk 18]
```diff
@@ -143,24 +143,24 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
 
 + 是否使用统一的读取线程池（在 [`readpool.unified`](#readpoolunified) 中配置）处理存储请求。该选项值为 false 时，使用单独的存储线程池。通过本节 (`readpool.storage`) 中的其余配置项配置单独的线程池。
-+ 默认值：true
++ 默认值：如果本节 (`readpool.storage`) 中没有其他配置，默认为 true。否则，为了升级兼容性，默认为 false，请根据需要更改 [`readpool.unified`](#readpoolunified) 中的配置后再启用该选项。
 
 ### `high-concurrency`
 
 + 处理高优先级读请求的线程池线程数量。
-+ 默认值：4
++ 当 8 ≤ cpu num ≤ 16 时，默认值为 cpu_num * 0.5；当 cpu num < 8 时，默认值为 4；当 cpu num > 16 时，默认值为 8。
```
Comment (TomShawn) [inline tikv-configuration-file.md:151]:
```suggestion
+ 当 8 ≤ cpu num ≤ 16 时，默认值为 cpu_num * 0.5；当 cpu num \< 8 时，默认值为 4；当 cpu num > 16 时，默认值为 8。
```

File: tikv-configuration-file.md [hunk 19]
```diff
@@ -143,24 +143,24 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
 + 最小值：1
 
 ### `normal-concurrency`
 
 + 处理普通优先级读请求的线程池线程数量。
-+ 默认值：4
++ 当 8 ≤ cpu num ≤ 16 时，默认值为 cpu_num * 0.5；当 cpu num < 8 时，默认值为 4；当 cpu num > 16 时，默认值为 8。
```
Comment (TomShawn) [inline tikv-configuration-file.md:157]:
```suggestion
+ 当 8 ≤ cpu num ≤ 16 时，默认值为 cpu_num * 0.5；当 cpu num \< 8 时，默认值为 4；当 cpu num \> 16 时，默认值为 8。
```

File: tikv-configuration-file.md [hunk 20]
```diff
@@ -143,24 +143,24 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
 + 最小值：1
 
 ### `low-concurrency`
 
 + 处理低优先级读请求的线程池线程数量。
-+ 默认值：4
++ 当 8 ≤ cpu num ≤ 16 时，默认值为 cpu_num * 0.5；当 cpu num < 8 时，默认值为 4；当 cpu num > 16 时，默认值为 8。
```
Comment (TomShawn) [inline tikv-configuration-file.md:163]:
```suggestion
+ 当 8 ≤ cpu num ≤ 16 时，默认值为 cpu_num * 0.5；当 cpu num \< 8 时，默认值为 4；当 cpu num \> 16 时，默认值为 8。
```

**Review by TomShawn — COMMENTED** [review]
To fix the CI failure

File: tikv-configuration-file.md (hunk 2 above)
Comment (TomShawn) [reply tikv-configuration-file.md:49]: PTAL: https://github.com/pingcap/docs-cn/pull/5886
